files/ko/web/api/htmlformelement/elements/index.html

```diff
@@ -0,0 +1,111 @@
+
+<p>{{domxref("HTMLFormElement.length")}} 속성을 독립적으로 사용하면 컨트롤 폼의 수만 얻을 수 있습니다.</p>
+
+<p>인덱스나 요소의 <code>name</code> 또는 <code>id</code> 속성을 사용하여 반환된 컬렉션의 특정 컨트롤 폼에 액세스할 수 있습니다.</p>
+
+<p>HTML 5 이전에는 <code>HTMLFormControlsCollection</code>을 기반으로 하는 {{domxref("HTMLCollection")}}을 반환하였습니다.</p>
+
+<div class="blockIndicator note">
+    <p><strong>참고:</strong> 문서의 {{HTMLElement("form")}} 속성을 사용해도 마찬가지로 주어진 문서에 포함된 모든 컨트롤 폼의 목록을 얻을 수 있습니다.</p>
+</div>
+
+<p>{{InheritanceDiagram(600,120)}}</p>
+
+<h2 id="Value">값</h2>
+
+<p>이미지가 아닌 모든 컨트롤을 포함한 {{domxref("HTMLFormControlsCollection")}}를 가집니다. 이것은 라이브 컬렉션입니다. 컨트롤 폼이 컬렉션에 추가되거나 제거되면 이 컬렉션은 변경 사항을 반영하도록 업데이트됩니다.</p>
+
+<p>반환된 컬렉션의 컨트롤 폼은 사전 순서, 깊이 우선 트리 순회를 따라 양식에 나타나는 순서와 동일합니다. 이것을 <strong>트리 오더</strong>라고 합니다.</p>
+
+<p>다음 요소들만 반환합니다:</p>
+
+<ul>
+    <li>{{HTMLElement("button")}}</li>
+    <li>{{HTMLElement("fieldset")}}</li>
+    <li>{{HTMLElement("input")}} (단, {{htmlattrxref("type", "input")}}이 <code>"image"</code>인 경우는 역사적으로 다른 기능을 위해 작동했기 때문에 생략됩니다.)</li>
+    <li>{{HTMLElement("object")}}</li>
+    <li>{{HTMLElement("output")}}</li>
+    <li>{{HTMLElement("select")}}</li>
+    <li>{{HTMLElement("textarea")}}</li>
+</ul>
+
+<h2 id="Examples">예</h2>
+
+<h3 id="quick_syntax_example">빠른 구문 예제</h3>
+
+<p>이 예제에서는 컨트롤 폼 목록을 얻는 방법과 인덱스 및 이름 또는 ID별로 해당 요소에 액세스하는 방법을 알아봅니다.</p>
+
+<pre class="brush: html">&lt;form id="my-form"&gt;
+  &lt;input type="text" name="username"&gt;
+  &lt;input type="text" name="full-name"&gt;
+  &lt;input type="password" name="password"&gt;
+&lt;/form&gt;</pre>
+
+<pre class="brush: js">var inputs = document.getElementById("my-form").elements;
+var inputByIndex = inputs[0];
+var inputByName = inputs["username"];</pre>
+
+<h3 id="accessing_form_controls">컨트롤 폼에 액세스하기</h3>
+
+<p>이 예제에서는 폼의 요소 목록을 가져온 다음 목록을 반복하면서 {{HTMLElement("input/text", "text")}} 유형의 {{HTMLElement("input")}} 요소를 찾아 일부 형식의 처리를 수행할 수 있도록 합니다.</p>
+
+<pre class="brush: js">var inputs = document.getElementById("my-form").elements;
+
+// Iterate over the form controls
+for (i = 0; i < inputs.length; i++) {
+  if (inputs[i].nodeName === "INPUT" && inputs[i].type === "text") {
+    // Update text input
+    inputs[i].value.toLocaleUpperCase();
+  }
+}</pre>
+
+<h3 id="disabling_form_controls">폼 컨트롤 비활성화</h3>
+
+<pre class="brush: js">var inputs = document.getElementById("my-form").elements;
+
+// Iterate over the form controls
+for (i = 0; i < inputs.length; i++) {
+  // Disable all form controls
+  inputs[i].setAttribute("disabled", "");
+}</pre>
+
+<h2 id="Specifications">명세</h2>
+
+<table class="standard-table">
+    <tbody>
+    <tr>
+        <th scope="col">명세</th>
+    </tr>
+    <tr>
+        <td>{{SpecName('HTML WHATWG', "forms.html#dom-form-elements-dev", "#dom-form-elements-dev")}}</td>
+    </tr>
+    </tbody>
+</table>
+
+<h2 id="Browser_compatibility">Browser compatibility</h2>
+
+
+
+<p>{{Compat("api.HTMLFormElement")}}</p>
+
+<h2 id="See_also">See also</h2>
```

Comment:
```suggestion
<h2 id="See_also">같이 보기</h2>
```

https://github.com/mdn/translated-content/blob/main/docs/ko/translation-guide.md#section-title 에 따라 `같이보기`로 번역하면 좋을것같아 리뷰드렸습니다.